source/common/buffer/buffer_impl.cc

```diff
@@ -554,6 +554,8 @@ OwnedImpl::OwnedImpl(const Instance& data) : OwnedImpl() { add(data); }
 
 OwnedImpl::OwnedImpl(const void* data, uint64_t size) : OwnedImpl() { add(data, size); }
 
+OwnedImpl::OwnedImpl(BufferMemoryAccountSharedPtr account) : OwnedImpl() { bindAccount(account); }
```

Comment:
You shouldn't call virtual methods from a constructor unless they are marked final.  Consider setting account_ directly instead.

Reply:
Done.